logback-contrib/README.md

```diff
@@ -0,0 +1,18 @@
+# logback-contrib  [![GitHub release](https://img.shields.io/github/release/qos-ch/logback-contrib.svg?maxAge=2592000)]() [![Build Status](https://travis-ci.org/qos-ch/logback-contrib.svg?branch=master)](https://travis-ci.org/qos-ch/logback-contrib)
+
+
+https://github.com/qos-ch/logback-contrib/wiki
+
+#### Build Instructions
+Run the following command to build:
+
+```
+mvn clean install
+```
+
+#### License
+```
+This program and the accompanying materials are licensed under
+the terms of the Eclipse Public License v1.0 as published by
+the Eclipse Foundation.
+```
```

Comment:
```suggestion
#### License
```
This maven module subtree and the accompanying materials are licensed under
the terms of the Eclipse Public License v1.0 as published by
the Eclipse Foundation.
```
```